monthandyearpicker/src/main/java/com/whiteelephant/monthpicker/MonthPickerDialog.java

```diff
@@ -179,6 +179,8 @@ private void showYearOnly() {
         _monthPicker.showYearOnly();
     }
 
+    private void setReverseYearOrder(Boolean reverseOrder) { _monthPicker.setReverseYearOrder(reverseOrder); }

```

Comment:
Primitive boolean can be used instead.
https://stackoverflow.com/questions/3728616/boolean-vs-boolean-in-java

Reply:
done